onnx/test/shape_inference_test.py

```diff
@@ -705,6 +835,10 @@ def test_reshape_static_shape_allowzero(self, _, version) -> None:
 
     @parameterized.expand(all_versions_for("Reshape"))
     def test_reshape_static_shape_constant(self, _, version) -> None:
+        if version == 1:
```

Comment:
Is this just version 1? Will we be able to ignore it to reduce much of the special case handling here?